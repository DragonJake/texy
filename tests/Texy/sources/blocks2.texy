/---html

space

\---

/---text


space


\---

/--.[texy]
te
st
\--

/-- .[texy]
te
st
\--

/--
0
\--

/--code .(0)
0
\--

/--code html .(0)
te
st
\--

/-- code html
te
st
\--



/--text any.text*123(a)4[5]679čřýřžáý
Text mezi @notexy ..... a @ nebude Texy *formátovat*
\--any.text*123(a)4[5]679čřýřžáý


/--texysource
Text mezi @notexy ..... a @ nebude Texy *formátovat*
\--

/--code
  function reImage($matches) {
    $content = $matches[1];
    $align = $matches[5];
    $href = $matches[6];
  }
  &nbsp; <br />
  &amp;amp;
\-- xxx

/--html
Text mezi @notexy ..... a @ nebude Texy *formátovat*
  &nbsp; <br />
\--

/--text
Text mezi @notexy ..... a @ nebude Texy *formátovat*
  &nbsp; <br />
\--
