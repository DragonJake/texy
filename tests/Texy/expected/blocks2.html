space space
<p>/--.<a href="http://texy.info/" title="The best text  → HTML converter and formatter">Texy!</a> te st \--</p>

<pre class="texy">te
st</pre>

<pre>0</pre>

<pre title="0"><code>0</code></pre>

<pre title="0" class="html"><code>te
st</code></pre>

<pre class="html"><code>te
st</code></pre>
Text mezi @notexy ..... a @ nebude Texy *formátovat*
<pre class="html"><code>&lt;p&gt;Text mezi @notexy ..... a @ nebude Texy &lt;em&gt;formátovat&lt;/em&gt;&lt;/p&gt;</code></pre>

<pre><code>function reImage($matches) {
  $content = $matches[1];
  $align = $matches[5];
  $href = $matches[6];
}
&amp;nbsp; &lt;br /&gt;
&amp;amp;amp;</code></pre>
Text mezi @notexy ..... a @ nebude Texy *formátovat*  <br />
Text mezi @notexy ..... a @ nebude Texy *formátovat*<br />
&amp;nbsp; &lt;br /&gt;
